classes/controller/AdminController.php

```diff
@@ -1931,6 +1932,20 @@ public function display()
             if (!is_array($this->$type)) {
                 $this->$type = (array) $this->$type;
             }
+
+            if (
+                SymfonyContainer::getInstance()
+                    ?->get('request_stack')
+                    ->getCurrentRequest()
+                    ->query->has('symfony_layout')
+            ) {
+                /** @var AddFlashMessage $addFlashMessage */
+                $addFlashMessage = SymfonyContainer::getInstance()->get(AddFlashMessage::class);
+                foreach ($this->$type as $message) {
+                    $addFlashMessage($type, array_shift($this->$type));
```

Comment:
```suggestion
                    $addFlashMessage($type, $message);
```
Why not use the loop variable directly? Is it because you also want to employ the array while you loop through it?

If yes, I don't doubt it works but I'm always uncomfortable modifying an array while I'm looping through it, it's kind of like sawing the branch you're standing on 😅 Isn't it possible to loop "regularly" and use `$message` and end after the loop with a `$this->$type = []`

Reply:
As you want :)